packages/admin/config/filament.php

```diff
@@ -207,10 +207,12 @@
         ],
         'sidebar' => [
             'is_collapsible_on_desktop' => false,
+            'hamburger_icon_on_desktop' => false,
             'groups' => [
                 'are_collapsible' => true,
             ],
-            'width' => null,
+            'width' => '20rem',
```

Comment:
Lets keep the `width` & `collapsed_width ` defaults in the views instead, because then we can change them later if we want without making people republish configs

Reply:
I understand. 
My thought, when adding the default value to config, was that we publish the config when we want to change the styling to ... whatever, and when doing so, we don't have to dig into the code to see what the current default value is?
I don't mind keeping it as `null`, just commenting on why I made the change.